runtime/doc/spell.txt

```diff
@@ -563,9 +563,7 @@ When the Myspell files are updated you can merge the differences:
 
 SPELL FILE VERSIONS					*E770* *E771* *E772*
 
-Spell checking is a relatively new feature in Vim, thus it's possible that the
-.spl file format will be changed to support more languages.  Vim will check
-the validity of the spell file and report anything wrong.
+Vim will check the validity of the spell file and report anything wrong.
```

Comment:
the sentence now lacks context. Before, "anything wrong" was in the context of "potential future format changes".

Reply:
And instead of requesting a change, you close the PR :+1:

Reply:
@nkakouros 
Feel free to re-open, of course.
I suggest though proposing the change to Vim directly.